cmd/minishift/cmd/delete.go

```diff
@@ -98,6 +100,28 @@ func removeInstanceConfigs() {
 	}
 }
 
+func removeProfileFromConfig() {
+	name := profileActions.GetProfileName()
+	activeProfile := profileActions.GetProfileName()
+
+	//If there are multiple profiles present
+	if name == activeProfile {
+		if !forceMachineDeletion {
+			fmt.Print("\nDo you want to delete the active profile? [y/N]: ")
+			fmt.Scanln(&confirm)
```

Comment:
we have code to ask for confirmation: AskForConfirmation(). For example, see `registration.go`